viz-lib/src/visualizations/table/renderer.less

```
@@ -12,6 +12,20 @@
     overflow-x: auto;
   }
 
+  .table-fixed-header {
+    table {
+      border-top: 0;
+
+      th {
+        position: sticky !important;
+        left: 0;
+        top: 0;
+        border-top: 0px solid #e8e8e8;
```

```suggestion
        border-top: hidden;
```

Let's have it as `border-top: hidden` and I will add the border in another PR only in the Redash repository (v9). Sounds good to you?

---

> Let's have it as `border-top: hidden` and I will add the border in another PR only in the Redash repository (v9) to avoid any conflicts. Sounds good to you?

Ok, i will use this PR. Thanks a lot.